api/test_runner_test.go

```diff
@@ -57,6 +58,7 @@ func NewAPITestRunner(t *testing.T) *APITestRunner {
 
 func (r *APITestRunner) Delete() {
 	os.RemoveAll(r.TmpDir)
+	os.RemoveAll("/tmp/api_test")
```

Comment:
This is a bad hack to make tests work as the JSON suite is hard coded and the tmpdir is generated at runtime. Maybe invest a bit of time to make some sort of templating of values possible in the suite. I'm just afraid of introducing "special logic" to the test file contents which would make the tests much less portable and require other users to understand implementation details of our own.

Reply:
totally, I'd ask that we make this directory `/tmp/qri_api_test` just to be super sure the dir doesn't collide with other software